Assets/HoloToolkit-Examples/Throwing/Scripts/ThrowingExampleSceneManager.cs

```diff
@@ -0,0 +1,170 @@
+﻿using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.XR.WSA.Input;
+
+public class ThrowingExampleSceneManager : MonoBehaviour
+{
+    public GameObject[] ThrowingAssets;
+    public InteractionSourceNode ControllerPose = InteractionSourceNode.Grip;
+    public Transform RealWorldRoot;
+
+    private readonly Dictionary<uint, Transform> devices = new Dictionary<uint, Transform>();
+    private readonly Dictionary<uint, int> modelIndecies = new Dictionary<uint, int>();
+    private readonly Dictionary<uint, bool> isDetatched = new Dictionary<uint, bool>();
+
+    // Use this for initialization
+    void Start ()
+    {
+		if (ThrowingAssets.Length == 0)
+        {
+            throw new System.Exception("ThrowingSceneManager needs to have some throwable objects!");
+        }   
+
+        InteractionManager.InteractionSourceReleased += InteractionManager_InteractionSourceReleased;
```

Comment:
We should probably be using the input manager to get our interactions.

Reply:
I want to minimize dependencies, as this is just a sample, allowing users to add throwing without needing to dig through the whole input stack

Reply:
If that's the case, then should we have the Examples in the main repository?

Reply:
I don't think so.  This scene is just a touch of code showing how one should consume controller information to take advantage of throwing.  Its sample code, not a component to use elsewhere.